docs-2.0/k8s-operator/4.cluster-administration/4.7.security/4.7.1.enable-mtls.md

```diff
@@ -493,34 +502,92 @@ Details about these fields are as follows:
 
 ### Configure `sslCerts`
 
-When you enable mTLS between services, you still needs to set `spec.sslCerts`, because NebulaGraph Operator communicates with the Meta service and Storage service. 
+As NebulaGraph Operator needs to communicate with the Meta and Storage services, you must configure `spec.sslCerts` to set certificates for it when enabling mTLS between services.
 
-The `spec.sslCerts` field specifies the encrypted certificates for NebulaGraph Operator and the [nebula-agent](https://github.com/vesoft-inc/nebula-agent) client (if you do not use the default nebula-agent image in Operator).
+The `spec.sslCerts` field specifies the encryption certificates for NebulaGraph Operator and [nebula-agent](https://github.com/vesoft-inc/nebula-agent) (if you do not use the default nebula-agent image).
 
-```yaml
-spec:
-  sslCerts:
-    clientSecret: "client-cert" 
-    clientCert: "tls.crt"  # The key name of the certificate in the client certificate Secret, default is "tls.crt".
-    clientKey: "tls.key"   # The key name of the private key in the client certificate Secret, default is "tls.key".
-    caSecret: "ca-cert" # The name of the CA certificate Secret.
-    caCert: "tls.crt"  # The key name of the certificate in the CA certificate Secret, default is "tls.crt".
-```
+You can create a Secret to pass in the certificates or configure the environment variables of NebulaGraph Operator to obtain local certificates.
```

Comment:
Please make it clear that when environment variable are used, the secrets under the `spec.sslCerts` section will be ignored.

Reply:
done